src/main/java/com/cloudogu/scm/ci/cistatus/service/CIStatusService.java

```diff
@@ -41,20 +41,23 @@ public CIStatusService(DataStoreFactory dataStoreFactory) {
     this.dataStoreFactory = dataStoreFactory;
   }
 
-  public void put(Repository repository, String changesetId, CIStatus ciStatus) {
+  public void put(String storeName, Repository repository, String changesetId, CIStatus ciStatus) {
```

Comment:
For the time being I would use an enum for the store (with a text field for the concrete name), so that one cannot mistakenly use other strings here.